multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/FlowableConfiguration.java

```diff
@@ -34,6 +36,8 @@
 
     private static final int ASYNC_JOB_ACQUIRE_WAIT_TIME_IN_MILLIS = (int) TimeUnit.SECONDS.toMillis(3);
     private static final int JOB_EXECUTOR_LOCK_TIME_IN_MILLIS = (int) TimeUnit.MINUTES.toMillis(30);
+    private static final long JOB_CANCEL_WAIT_TIME_IN_MILLIS = TimeUnit.MINUTES.toMillis(1);
```

Comment:
Is this the time between the submission of the job to the job executor and the starting of its execution?

Reply:
this is the sleep period for `CancelExpiredTasksThread`. I wanted it to sound more like the other variables, but maybe I should rename it to `CANCEL_EXPIRED_TASKS_THREAD_SLEEP_TIME_IN_MILLIS` or something like that